flake.nix

```diff
@@ -97,6 +97,8 @@
                   src = inputs.holochain;
                   filter = includeFilesFilter;
                 };
+                # Do release builds because they run faster
+                CARGO_PROFILE = "release";
```

Comment:
Where does this get used? I don't see a reference to it :slightly_smiling_face:

Reply:
I suppose this is injected as described here https://crane.dev/API.html?highlight=cargo_profile#cranelibcargohelperfunctionshook

But I can't tell from the workflow output.

Reply:
Ah, okay. Thanks

Reply:
After some more experimentation, I see we don't actually have to set this - it defaults if not set https://crane.dev/API.html?highlight=cargo_profile#cranelibconfigurecargocommonvarshook